Tasks/Common/ssm.ts

```diff
@@ -8,6 +8,7 @@ import tl = require('vsts-task-lib/task')
 export const transformCustom = 'custom'
 export const transformSubstitute = 'substitute'
 export const readModeSingle = 'single'
+export const readModeHierarchy = 'hierarchy'
```

Comment:
`readModeHierarchy` doesn't appear to be used anywhere

Reply:
it's used in a subsequent PR